index.html

```diff
@@ -5,8 +5,165 @@
     <title>HTML Academy: Седона</title>
   </head>
   <body>
+    <!--Header-->
+    <header class="page-header">
+      <!--Навигация-->
+      <nav class="navigation">
+        <!--Логотип-->
+        <a href="index.html">
+          <img src="" alt="Логотип Седоны">
+        </a>
+        <!--Основная навигация по сайту-->
+        <ul class="navigation-list">
+          <li class="navigation-item active">
+            <a class="navigation-link" href="index.html">Главная</a>
```

Comment:
Когда есть ссылка на текущую страницу в навигации, это немного смущает. 

Значит либо мы убираем `href`
```suggestion
            <a class="navigation-link">Главная</a>
```

Либо ставим нужный [`aria` атбрибут](https://www.aditus.io/aria/aria-current/)
```suggestion
            <a class="navigation-link" href="index.html" aria-current="page">Главная</a>
```